ansible/roles/hyperthreading/files/disable-hyperthreading.sh

```diff
@@ -24,11 +24,12 @@ fi
 # /sys/devices/system/cpu/cpu0/topology/thread_siblings_list:0
 # /sys/devices/system/cpu/cpu1/topology/thread_siblings_list:1
 
-SIBLING_COUNT=`grep -H . /sys/devices/system/cpu/cpu*/topology/thread_siblings_list | sort -n -t ':' -k 2 -u | grep "," | wc -l`
+# note if there are no thread_siblings_list the grep will error, but wc will count 0 and be successful.  this is a valid result.
+SIBLING_COUNT=`grep '[^0-9]' /sys/devices/system/cpu/cpu*/topology/thread_siblings_list | wc -l`
 if [ "$SIBLING_COUNT" == 0 ]; then
     exit 0
 else  
-    echo "Found this many cpu's with siblings: $SIBLING_COUNT"
-    echo "Expected no siblings!"
+    echo "Found this many in thread_siblings_list: $SIBLING_COUNT"
```

Comment:
Since "sort ... -u" is removed, the count will be generally double (one line for each CPU).  No need to fix, but it's mildly vague "found this many whats?" in the phrasing.